vtex-apps/pwa-gtm-pixel
# Init app

#### What is the purpose of this pull request?
Init app and sent `installWebApp` event to google tag manager `dataLayer`.

#### What problem is this solving?
We need to measure and show how much PWA features contributed to the overall user experience on VTEX stores.

#### How should this be manually tested?
[Workspace](https://pwaevents2--storecomponents.myvtex.com/checkout/orderPlaced/?og=949500478939)
1. Click 'install' on install banner at the bottom of the page
2. Choose the option you desire on the A2HS prompt
3. Open the console, type `dataLayer` and check if the event was added

#### Types of changes
